### PR TITLE
new package: keychain 2.8.5

### PR DESCRIPTION
**Pull request by a-schaefers**
## Description of package
a nice little tool written by the original creator of Gentoo for handling gpg and ssh passphrases :)
## Checklist

- [x] Latest upstream version.
- [x] I agree to maintain this package (**required**).
- [x] I agree to notify KISS if I can no longer maintain this package (**required**).

### CODE REVIEW

File: community/keychain/build [hunk 1]
```diff
@@ -0,0 +1,7 @@
+#!/bin/sh -e
+
+
+
```
Comment (dylanaraps) [inline community/keychain/build]: Remove these two blank lines.

File: community/keychain/build [hunk 2]
```diff
@@ -0,0 +1,7 @@
...
+mkdir -p "$1/usr/bin"
```
Comment (dylanaraps) [inline community/keychain/build]: Remove this line.

File: community/keychain/build [hunk 3]
```diff
@@ -0,0 +1,7 @@
...
+
+install -m755 "mksh" "$1/usr/bin/mksh"
```
Comment (dylanaraps) [inline community/keychain/build]:
```
 install -Dm755 "mksh" "$1/usr/bin/mksh"
```

Add `-D` and also fix this as it is from `mksh`.

Comment (a-schaefers) [reply community/keychain/build]: lol. must have needed moar coffee when doing this one eh. fixed

**Pull request merged**